controls/radgridview/events/column-resize-event.md

```diff
@@ -46,19 +46,19 @@ You can subscribe to the events either declaratively or runtime. **Example 1** s
 
 ## AutoGeneratingColumn
 
-When RadGridView's **AutoGenerateColumns** property is set to **True** (the default value), RadGridView will create a separate column for each public property of the bound objects and the **AutoGeneratingColumn** event will be triggered for each of those columns.
+When you set RadGridView's **AutoGenerateColumn** property to **True** (the default value), RadGridView creates a column for each public property of the bound objects and an **AutoGeneratingColumn** event is  triggered for each of those columns.
```

Comment:
Bear in mind that there's only one **AutoGeneratingColumn** event (the same applies to all other events) so the definite article (the) should be used (in my opinion).